atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraServersConfigs.java

```diff
@@ -111,14 +118,16 @@ final void check() {
         }
 
         @Override
-        public final <T> T visit(Visitor<T> visitor) {
-            return visitor.apply(
-                    hosts().stream()
-                            .map(host -> new InetSocketAddress(host, thriftPort()))
-                            .collect(Collectors.toSet()),
-                    Optional.of(hosts().stream()
-                            .map(host -> new InetSocketAddress(host, cqlPort()))
-                            .collect(Collectors.toSet())));
+        public <T> T visitThrift(Function<Set<InetSocketAddress>, T> visitor) {
+            return visitor.apply(hosts().stream()
+                    .map(host -> InetSocketAddress.createUnresolved(host, thriftPort()))
```

Comment:
why unresolved here?

Reply:
Did not rebase on current develop. Fixed.